connect/client.go

```diff
@@ -240,6 +240,14 @@ func (rs *restClient) GetItemsByTitle(title string, vaultUUID string) ([]onepass
 		return nil, err
 	}
 
+	for i := 0; i < len(items); i++ {
```

Comment:
Not a necessary change but we can do a more clean way of looping through the items in Golang:

```for index, item := range items {
    tempItem, err := rs.GetItem(item.ID, item.Vault.ID)
    if err != nil {
			return nil, err
		}
		item = *tempItem
}```

Reply:
I believe your comment may have a typo, but I tried to "style" this loop a bit, have a look and let me know if this is what you meant. :D

Reply:
Ping. This is still a problem.

Reply:
With the latest changes this should be solved now.